### PR TITLE
Update and test existing documentation snippets

### PR DESCRIPTION
**Pull request by KPrasch**
Note, this does not include the Readme, or the setting of default curves.

### CODE REVIEW

File: docs/source/using_pyumbral.rst [hunk 1]
```diff
@@ -48,27 +77,51 @@ Invocation of `umbral.encrypt` returns both the `ciphertext`,
 and a `capsule`, Anyone with Alice's public key can perform
```
Comment (cygnusv) [inline docs/source/using_pyumbral.rst]: `umbral.encrypt` --> `pre.encrypt`

File: docs/source/using_pyumbral.rst [hunk 2]
```diff
@@ -80,14 +133,19 @@ which are next sent to N proxies or *Ursulas*.
 | `M` - Minimum threshold of key fragments needed to activate a capsule.
```
Comment (cygnusv) [inline docs/source/using_pyumbral.rst]: `M` --> `threshold`?

File: docs/source/using_pyumbral.rst [hunk 3]
```diff
@@ -96,26 +154,20 @@ S3, IPFS, Google Cloud, Sneakernet, etc.
...
 .. code-block:: python
 
-   # Generate a key pair for Bob
-   bobs_private_key = keys.UmbralPrivateKey.gen_key()
-   bobs_public_key = private_key.get_pubkey()
-
-   # Bob receives the capsule
-   capsule = <fetch a capsule through side channel>
+   # Bob receives the capsule through a side-channel
+   capsule = capsule
```
Comment (cygnusv) [inline docs/source/using_pyumbral.rst]: Can we do the `bob_capsule = capsule` thing?

File: docs/source/using_pyumbral.rst [hunk 4]
```diff
@@ -132,26 +184,47 @@ Bob collects the resulting `cfrags` from several Ursulas.
 Bob must gather at least `M` `cfrags` in order to activate the capsule.
```
Comment (cygnusv) [inline docs/source/using_pyumbral.rst]: `M` here (and in line 180) perhaps should be renamed to `threshold`.

File: docs/source/using_pyumbral.rst [hunk 5]
```diff
@@ -132,26 +184,47 @@ Bob collects the resulting `cfrags` from several Ursulas.
...
-      cfrags.append(cfrag)    # Bob collects a cfrag
+    cfrags = list()                   # Bob's cfrag collection
+    for kfrag in kfrags:
+      cfrag = pre.reencrypt(kfrag=kfrag, capsule=capsule)
+      cfrags.append(cfrag)            # Bob collects a cfrag
 
 
 Bob attaches cfrags to the capsule
 ----------------------------------
 Bob attaches at least `M` `cfrags` to the capsule;
```
Comment (cygnusv) [inline docs/source/using_pyumbral.rst]: `M` again.

**Comment by KPrasch** [comment]
Thanks for the feedback! After building the HTML docs, I noticed that using both doctest and code blocks together result in duplicated code snippets, So I'm changing this to a WIP until I refactor.

**Pull request merged**